src/internal/svg-icons/Add.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import pure from 'recompose/pure';
+import SvgIcon from '../../SvgIcon/index';
+
+/**
+ * @ignore - internal component.
+ */
+let Add = props => (
+  <SvgIcon {...props}>
+    <path d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z" />
+  </SvgIcon>
+);
+Add = pure(Add);
+Add.muiName = 'SvgIcon';
+
+export default Add;
```

Comment:
Unrelated to this PR, but would it be possible to import those icons from `material-ui-icons` instead of copying them?

Reply:
We'd end up with a circular dependancy (material-ui-icons depends on material-uo/SvgIcon), so not sure how that would work out in practice.

Reply:
Oh, you're right. I guess this would be possible by `npm link`ing `packages/material-ui` into `packages/material-ui-icons` (lerna/lerna might even do this automagically, but I'm not sure). Maybe we should keep it as is then.

Reply:
yarn workspace is another option.